Library/Formula/freeswitch.rb

```diff
@@ -19,20 +19,24 @@ class Freeswitch < Formula
   depends_on "pkg-config" => :build
   depends_on :apr => :build
 
-  depends_on "jpeg"
+  option "without-moh", "Do not install music-on-hold"
+  option "without-sounds-en", "Do not install English (Callie) sounds"
+  option "with-sounds-fr", "Install French (June) sounds"
+  option "with-sounds-ru", "Install Russian (Elena) sounds"
+
   depends_on "curl"
+  depends_on "jpeg"
+  depends_on "ldns"
   depends_on "openssl"
   depends_on "pcre"
   depends_on "speex"
   depends_on "sqlite"
 
   def install
-    system "./bootstrap.sh", "-j#{ENV.make_jobs}"
+    system "./bootstrap.sh", "-j"
```

Comment:
When I ran `./bootstrap.sh -j8` manually, I got an error message saying that the option wasn't recognized. When I looked at the script, it didn't look like it was expecting a number after `-j` at all. Running `./bootstrap.sh -j` worked and gave a measurable improvement over plain `./bootstrap.sh`.